package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "schemats",
-  "version": "3.0.7-embark",
```

Comment:
Being semantically pedantic this should be `3.0.7-embark-2` unless we've updated against the source library's `3.0.8`

Reply:
The original schemats is 3.0.3, this seems to be what has been done to bump the version before 🤷

Reply:
Looks like the original (https://github.com/SweetIQ/schemats) is abandonware, so TBH we should probably start numbering it ourselves.

Reply:
In the past we considered taking over this repo and publishing updates to it so we tried to keep the version in line with what would be used in the library. We never did this but have kept the same versioning. The `-embark` now just denotes it is our version.